lib/switchtec.c

```diff
@@ -960,7 +1068,7 @@ static int log_a_to_file(struct switchtec_dev *dev, int sub_cmd_id,
 
 	if (log_def_file != NULL) {
 		/* read the log definition file into defs */
-		ret = read_log_defs(log_def_file, &defs);
+		ret = read_log_defs(log_def_file, true, &defs);
```

Comment:
Not withstanding that I think we should drop this function; but using "true" here is very confusing.

Reply:
Right -- originally the parameter was a bool, but I updated it to the enum and forgot to change this.